services/visual-studio-marketplace/visual-studio-marketplace-rating.service.js

```diff
@@ -63,20 +63,10 @@ module.exports = class VisualStudioMarketplaceRating extends VisualStudioMarketp
   async handle({ format, extensionId }) {
     const json = await this.fetch({ extensionId })
     const { statistics } = this.transformStatistics({ json })
-    const { value: averageRating } = this.getStatistic({
-      statistics,
-      statisticName: 'averagerating',
+    return this.constructor.render({
+      format,
+      averageRating: statistics.averagerating,
+      ratingCount: statistics.ratingcount,
     })
-
-    if (format === 'stars') {
-      return this.constructor.render({ format, averageRating })
-    }
-
-    const { value: ratingCount } = this.getStatistic({
-      statistics,
-      statisticName: 'ratingcount',
-    })
-
-    return this.constructor.render({ format, averageRating, ratingCount })
```

Comment:
This is definitely clearer!

Reply:
+12937123!

This flow definitely looks better (in all services), glad we ended up with this approach!